README.md

```diff
@@ -184,21 +177,29 @@ async fn main() -> anyhow::Result<()> {
 _Example output_
 
 ```json
```

Comment:
If we want to display JSON then we have to make the output JSON, otherwise it's weirdly formatted. You can remove the "Published DID document ..." part I think.